Cannot find module 'inquirer' when input name for Slack App

**Describe the bug**
Cannot find module 'inquirer' when run `amplify console-notification add`

**To Reproduce**
Steps to reproduce the behavior:
1. Run `amplify console-notification add`
2. Input name for Slack App
3. See error

**Expected behavior**
No error

**Screenshots**
<img width="828" alt="スクリーンショット 2022-12-15 8 59 50" src="https://user-images.githubusercontent.com/1638848/207741221-abb46a87-9780-4620-b0d7-d7f231c697f9.png">



**Desktop (please complete the following information):**
 - OS: Linux

--- a/README.md
+++ b/README.md
@@ -55,6 +55,7 @@ Later, input your Incoming Webhook URL when prompt to input webhook from amplify
 2. Enable this plugin
    ```sh
    amplify plugin add $(npm root -g)/amplify-category-console-notification
+   amplify plugin add $(npm root -g)/amplify-slack-nodejs-function-template-provider
    ```
 
 <!-- USAGE EXAMPLES -->

--- a/packages/amplify-slack-nodejs-function-template-provider/package.json
+++ b/packages/amplify-slack-nodejs-function-template-provider/package.json
@@ -27,7 +27,8 @@
   "dependencies": {
     "amplify-cli-core": "^2.4.14",
     "amplify-function-plugin-interface": "^1.9.4",
-    "enquirer": "^2.3.6"
+    "enquirer": "^2.3.6",
+    "inquirer": "^7.3.3"
   },
   "keywords": [
     "aws",

